docs/sources/panels/transformations.md

```diff
@@ -213,6 +204,19 @@ After I apply the transformation, my labels appear in the table as fields.
 
 {{< docs-imagebox img="/img/docs/transformations/labels-to-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+## Series to rows
+Use this transformation to combine the result from multiple time series data queries into one single result. This is helpful when using the table panel visualization. 
+
+The result from this transformation will contain three columns: `Time`, `Metric` and `Value`. The `Metric` column is added so you easily can see from what query the metric originate from. You can be customized this value by defining `Label` on the source query.
```

Comment:
Are you referring to an example here? Or will this be true in all cases? The way it is written, it seems true in all cases. If that is the case, then you're good.

Reply:
Yes, it is true for all cases